test/suite/hook_disagg.py

```diff
@@ -94,6 +94,9 @@ def wiredtiger_open_replace(orig_wiredtiger_open, homedir, curconfig):
     if 'tiered_storage=' in curconfig:
         skip_test("cannot run disagg hook on a test that uses tiered_storage in the config string")
 
+    if ('log=(enabled)') in curconfig:
```

Comment:
I see we have multiple tests with `log=(enabled)`. If we indeed are skipping it, then there is value in seeing what tests we can remove from `hook_disagg.fail`

Reply:
added more skip tests